angular-ui-router/angular-ui-router.d.ts

```diff
@@ -168,6 +168,17 @@ declare module angular.ui {
         current: IState;
         params: IStateParamsService;
         reload(): void;
+        
+        $current: IStateServiceUtilities;
+    }
+    
+    interface IStateServiceUtilities {
```

Comment:
I don't believe that either of these extends `IState`, based on the values on `$current`. I also know that neither `locals` nor `globals` looks similar to `IState`. I'm not sure what is being suggested, though, or I'd be happy to update my PR.